utils/lit/lit/formats/taef.py

```diff
@@ -38,6 +38,10 @@ def executeCommandForTaef(command, cwd=None, env=None):
 
 class TaefTest(TestFormat):
     def __init__(self, te_path, test_dll, test_path, select_filter, extra_params):
+        # Try to find te in %PATH% if not exist.
+        if not os.path.exists(te_path):
```

Comment:
Is it an error to provide a `te_path` that does not exist? Can we push the path search to the caller instead?

Reply:
That sounds a good idea.